docs/guide/getting-started.md

```diff
@@ -26,11 +26,51 @@ if (typeof window.ethereum !== 'undefined') {
+Navigate to https://faucet.metamask.io/
+Ensure you have the test account and test "Ropsten Test Network Selected"
+Click "reqest 1 ether from faucet"
+![image](https://user-images.githubusercontent.com/1038445/150909358-80beed6d-bebc-4602-90fb-183b15279228.png)
+
+A request will be sent to Metamask Client for you to sign. Click Next **WAIT Patiently for the transaction to be approved once Metamask has connected (may take a couple minutes)**
+![image](https://user-images.githubusercontent.com/1038445/150909613-6cb6d54e-3d43-4505-b8d6-7fa6b225e355.png)
+
+![image](https://user-images.githubusercontent.com/1038445/150909688-211f3398-e737-42db-bee2-e62d5e2ef5a8.png)
+
+![image](https://user-images.githubusercontent.com/1038445/150910232-703e1c26-4bdb-41aa-a174-7c10a9f3a6f8.png)
+
+![image](https://user-images.githubusercontent.com/1038445/150910318-b6cd9eb6-56e8-46b0-9664-82a072d3b0a5.png)
+
 ### Running a Test Network
 
-In the top right menu of MetaMask, select the network that you are currently connected to. Among several popular defaults, you'll find `Custom RPC` and `Localhost 8545`. These are both useful for connecting to a test blockchain, like [ganache](https://www.trufflesuite.com/ganache). You can quickly install and start Ganache if you have `npm` installed with `npm i -g ganache-cli && ganache-cli`.
+If you would like to run your own Network on you local machine. [Ganache](https://www.trufflesuite.com/ganache) has some great features for starting your application with different states.
+
+You can quickly install and start Ganache if you have `npm` installed with `npm i -g ganache-cli && ganache-cli`.
+Once Ganache is started connect to network `Localhost 8545` at the bottom of the network list (if you don't see `Localhost 8545` in your network list follow the instructions above [`Connecting to a test network`](#Connecting) to `show` test networks).
+You can now 
+![image](https://user-images.githubusercontent.com/1038445/150916135-c9d019ad-86f3-474d-a0ef-d53dabe799f9.png)
+
+If your application starts with the `-m` (bip39 mnemonic phrase for generating a PRNG seed) flag, you can feed it the same seed phrase you have in your MetaMask, and the test network will give each of your first 10 accounts 100 test ether, which makes it easier to start work.<br />
+Using the same Mnumonic phrase will generate the same set of 10 account numbers when you restart Ganache
+``` ganache-cli -m "random set of words to generate same account numbers each restart" ```
+**Example of a Mnemonic generated by Ganache**
 
-Ganache has some great features for starting your application with different states. If your application starts with the `-m` flag, you can feed it the same seed phrase you have in your MetaMask, and the test network will give each of your first 10 accounts 100 test ether, which makes it easier to start work.
+![image](https://user-images.githubusercontent.com/1038445/150917008-2a811b12-44f8-46b7-bf7b-b6907652fe31.png)
```

Comment:
I think we can make this section more concise. We advise that people use the same seed phrase in your MetaMask, and we say if you do this it will give your first 10 accounts 100 test ether. I don't think it's necessary to explain what `-m` does, as once you give an example it will become obvious. I also don't think it's necessary to say "make sure you use the same seed phrase" every time you start Ganache, unless we feel like the reader doesn't understand that a seed phrase is used to generate the account ids (if that's the case then maybe we could link to an article that explains more about BIP39). You will _need_ to use the same seed phrase that you've used to create accounts in MetaMask, otherwise your accounts in MetaMask won't have a balance.

Putting this together, what if we say:

``` suggestion
If you start `ganache-cli` with `-m` and provide the same seed phrase you've used to create your wallet in MetaMask, the first 10 accounts in your wallet will have 100 test ether.
```